b/boringssl/boringssl_ubuntu_18.04.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : boringssl
+# Version       : n/a
+# Source repo   : https://boringssl.googlesource.com/boringssl/
+# Tested on     : ubuntu_18.04
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Nishikant Thorat <Nishikant.Thorat@ibm.com>
+# Travis-Check  : True
+# Language	: go
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#
+# Install dependencies
+#
+apt-get install -y sudo
+sudo apt-get update -y
+sudo apt-get install -y cmake build-essential g++ wget git pkg-config libunwind-dev
+#
+# Install go
+#
+wget https://storage.googleapis.com/golang/go1.15.11.linux-ppc64le.tar.gz
+sudo tar -C /usr/local -xzf go1.15.11.linux-ppc64le.tar.gz
+export PATH=$PATH:/usr/local/go/bin
+rm go1.15.11.linux-ppc64le.tar.gz
+#
+# Clone and build source code
+#
+git clone https://boringssl.googlesource.com/boringssl/
+cd boringssl
+mkdir build
```

Comment:
Instead of taking the master latest commit, use the PACKAGE_VERSION variable to keep either version or commit id.

```
PACKAGE_VERSION=${1:-36a41bf}

git clone https://boringssl.googlesource.com/boringssl/
cd boringssl
git checkout $PACKAGE_VERSION
```

This make sure that, it build the master at 36a41bf commit if no parameter provided.
If you want to build a specific version, you can pass that version/commit id as command line parameter.